### DISCUSSION TITLE
[Resolved] "Bootleg" Flare on Steam store

### DISCUSSION THREAD

**Discussion by dorkster** [flareteam/flare-game#939] [General]
**EDIT: This has been resolved: https://github.com/flareteam/flare-game/discussions/939#discussioncomment-6698049**

It was brought to my attention that someone has uploaded Flare under the name [Undead World](https://store.steampowered.com/app/2193420/Undead_World/).

There's a [Steam forum thread](https://steamcommunity.com/app/2193420/discussions/0/3784751580939097946/) by someone who also noticed this.

While in bad taste IMHO, I think the GPL might allow for this. But it might be similar to the [0A.D. case](https://www.gamingonlinux.com/2022/10/someone-released-the-foss-rts-0-ad-on-steam-without-speaking-to-the-developers/), where it was more of an issue of the developer impersonating other developers.

I'm not sure if a DMCA claim would be the right option at the moment. I also don't see a way to contact the dev outside of the Steam forum. So maybe we should be vocal over there?

**Comment by Megalomaniak** [comment]
It's publisher is listed as the `Flare Engine Team` so it is giving some credit, albeit in a rather sneaky way. It's also not currently charged any money for and it's listed as early access.

It is possible someone is looking to do a genuine total conversion mod I suppose, but if so I do think they got ahead of themselves and put it up perhaps too soon(note: publishing on steam costs money, no free access to the platform, making it all the weirder). Or maybe there really is something nefarious going on, but I feel there is at least a chance it might not be entirely malicious.

**Comment by HoneSundan** [comment]
publisher have now become Undead World Team. 

What happened?

**Reply by dorkster** [reply]
No idea. I guess to avoid impersonating us. They *did* add this line on March 13 according to the [SteamDB history](https://steamdb.info/app/2193420/history/):

>This project is based on Flare engine, the soruce code and license remains same as engine GNU General Public License v3.0 and CC-BY-SA

And it is visible on the Steam page below the system requirements. Also, it's worth noting that, so far, the uploader has made no changes to the credits/readme files that are included in the depot.

One more interesting thing I noticed: they added depots of MSVC and DirectX for "compatibility". Guess they don't know that we compile the Windows version with MinGW and "force" OpenGL for rendering.

**Reply by HoneSundan** [reply]
https://steamcommunity.com/app/2193420/discussions/0/3784751580941131620/
![1](https://github.com/flareteam/flare-game/assets/83798843/36467a58-81ca-45c1-b126-60f91b980fff)
Maybe the problem has been solved?

**Reply by dorkster** [reply]
Thanks for posting Clint's update. I missed that.
